### ISSUE TITLE
this kind of errors should not be returned to the user. it should be said "we are working on it, your request will be ready after an intervention" since we are.

### ISSUE THREAD

**Issue by volodymyrss** [oda-hub/dispatcher-app#61]
from recent experience

**Comment by burnout87** [comment]
Is this happening in the situation of a `failed` request?

**Comment by volodymyrss** [comment]
yes, could we simply change the message? and be sure the informative message is sent to sentry?
it should be clear for user what to do with the information given by us.
As of now, often user will receive "unknown failure" and that's it. For me as a user, this would mean suggestion to leave and not come back.
Except when an error is of the kind like "request not understood" - "bad input parameters". I think we have separate exceptions for this already.

**Comment by burnout87** [comment]
It will be generated a message for the `failed` request case, containing the details the user might need.

**Comment by volodymyrss** [comment]
is it feasible to do quickly, considering also state of other features for the current milestone? it's ok if not!

**Comment by burnout87** [comment]
Depends on the set of information we want to include in the email. But I assume that everything is already there, so just a matter of preparing a different email format for this case. Should be not a big deal

**Comment by volodymyrss** [comment]
Indeed! If you want, you can also let us make a text, which might speak to our domain community. Especially if it is in a template or something. Did you use jinja2? It's good for templating.
But, this aspect can be left for later.

This sort of message can be also changed in respons on run_analysis.

**Comment by volodymyrss** [comment]
jinja2 is bundled with flask, so it's suitable here.

no problem, as far as I am concerned, this is a secondary issue for a future release.
I just note it since it is found in the wild.

**Comment by volodymyrss** [comment]
sorry if I am distracting from the flow! I did not mean there is any special need to track this problem now.

**Comment by burnout87** [comment]
No problem at all, it's good to keep this issue it in the loop, so we don't forget it. Could be something for the next release
